cmd/tracee-ebpf/main.go

```diff
@@ -87,12 +88,15 @@ func main() {
 			cfg.Output = &output
 
 			// environment capabilities
-
-			selfCap, err := getSelfCapabilities()
+			selfCap, err := capability.NewPid2(0)
```

Comment:
Why do we split capability.NewPid2(0) out of method that check capabilities of the current process? It seems that we always do capability.NewPid2 and then Load()

Reply:
I would keep the original implementation and usage of `getSelfCapabilities` and export it.

Reply:
I agree with you both. I did this to be able to dependency inject Load(). `NewPid2()` doesn't seem to be an a method defined through an interface so it's difficult to be able to dependency inject, unless we monkey patch it. 

Any other ways (besides the one suggested above) where we could keep `getSelfCapabilities` as it's today and also test the behaviour of both NewPid2() and Load() deterministically in a test? Thoughts?

Reply:
Please see the alternative approach as described above demonstrated here https://github.com/aquasecurity/tracee/pull/1256/commits/60d0a30163a9e5258d4bb427a49a65c61b1462c3